src/utils/feature.flag.ts

```diff
@@ -0,0 +1,12 @@
+import { getEnvironmentValueAsBoolean } from "./environment.value";
+
+type FeatureFlags = {
+    [key: string]: boolean
+}
+
+const flags: FeatureFlags = {
+  "FLAG_1": getEnvironmentValueAsBoolean("FLAG_1"),
+  "FLAG_2": getEnvironmentValueAsBoolean("FLAG_2")
+};
+
+export const hasFeature = (): FeatureFlags => flags;
```

Comment:
`hasFeature()` doesn't return a boolean but a list of feature flag boolean values, so that should be reflected in the function name.